src/main/scala/com/twitter/finagle/postgres/values/ValueEncoder.scala

```diff
@@ -211,3 +222,5 @@ object ValueEncoder extends LowPriorityEncoder {
 trait LowPriorityEncoder {
   implicit def fromExport[T](implicit export: ValueEncoder.Exported[T]): ValueEncoder[T] = export.encoder
 }
+
+case class JSONB(bytes: Array[Byte]) extends AnyVal
```

Comment:
Could this go in its own source file? I think we should try to refactor the custom data types into a separate package soon, but for now it should at least be in its own file for findability. `JSONB.scala`?

Reply:
Sure. It makes sense - I tried to keep the current organization.